Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/OS - Display Timeouts.json

```diff
@@ -8,7 +8,7 @@
       }
     }
   ],
-  "description": "Turn off display after 10 minutes when on battery and 20 minutes when plugged in.",
+  "description": "What is it: Turns off the display after 10 minutes when on battery and 20 minutes when plugged in.",
```

Comment:
**What does this do?**
Turns off the display after 10 minutes when running on battery and after 20 minutes when plugged in.

**Why should you use this?**
If you want to ensure devices save power and are inaccessible when the user is not present.

**What is the end-user impact?**
Users' device displays will turn off after 10 minutes when running on battery and after 20 minutes when plugged in.

**Learn more**
N/A

Reply:
yes this should be removed